build/linux/installer/conf/fluent-bit-network-flow-logs.conf

```diff
@@ -0,0 +1,30 @@
+[INPUT]
+    Name                tail
+    Alias               network_flow_input
+    Tag                 oms.networkflow.*
+    Path                /var/run/acns/hubble/events.log
+    Read_from_Head      true
+    DB                  /var/opt/microsoft/docker-cimprov/state/network_flow.db
+    DB.Sync             Off
+    Parser              json
+    # Mem_Buf_Limit       50m
+    # Buffer_Chunk_Size   2m # configurable
+    # Buffer_Max_Size     8m # configurable
+    Rotate_Wait         20
+    Refresh_Interval    5
+    storage.type        filesystem
```

Comment:
we would also need to configure https://docs.fluentbit.io/manual/administration/backpressure#storage.max_chunks_up. we would need bunch of high log scale config.

Reply:
storage.max_chunks_up is for service